packages/trip-form/src/TripOptions/styled.ts

```diff
@@ -7,6 +7,9 @@ import { isServerEnv } from "./util";
 // eslint-disable-next-line prettier/prettier
 import type { ButtonProps } from "./Checkbox"
 
+// todo: move this string to localization file (and possibly add more exact info on each particular mode)
+const modeButtonAriaLabel = "Opens a dialog that describes this mode, with optional links to third party services.";
```

Comment:
I worry it might be a pain to insert i18n strings from within the styled file. Is there a reason to not pass this string in via prop?

Reply:
good suggestion, Miles .. done.